mutagen-core/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(proc_macro_span)]
+#![feature(box_syntax)]
```

Comment:
Where is this needed?

Reply:
This is not strictly needed. This is a leftover from converting to boxed closures and getting those to work. I can revert this if necessary.

Reply:
Would be nice, otherwise we can remove it in a later PR. I usually try to minimize use of nightly features, that's why I asked.

Reply:
feature removed.